Index: Translate.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>weekdays = {'Понедельник': 'Monday',\r\n            'Вторник': 'Tuesday',\r\n            'Среда': 'Wednesday',\r\n            'Четверг': 'Thursday',\r\n            'Пятница': 'Friday',\r\n            'Суббота': 'Saturday',\r\n            'Воскресенье': 'Sunday',\r\n            }\r\nweather_effect =   {'облачно с прояснениями': 'cloudy with clearings',\r\n                    'сильный дождь': 'heavy rain',\r\n                    'переменная облачность': 'partly cloudy',\r\n                    'дождь': 'rain',\r\n                    'возможны грозы': 'thunderstorms possible',\r\n                    'мелкий дождь': 'light rain',\r\n                    'небольшая облачность': 'partly cloudy',\r\n                    'сплошная облачность': 'overcast',\r\n                    }
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Translate.py b/Translate.py
--- a/Translate.py	(revision 5b64cb32cf9887b1febb528b5df26a9bc3a987d4)
+++ b/Translate.py	(date 1660216927661)
@@ -6,12 +6,12 @@
             'Суббота': 'Saturday',
             'Воскресенье': 'Sunday',
             }
-weather_effect =   {'облачно с прояснениями': 'cloudy with clearings',
-                    'сильный дождь': 'heavy rain',
-                    'переменная облачность': 'partly cloudy',
-                    'дождь': 'rain',
-                    'возможны грозы': 'thunderstorms possible',
-                    'мелкий дождь': 'light rain',
-                    'небольшая облачность': 'partly cloudy',
-                    'сплошная облачность': 'overcast',
-                    }
\ No newline at end of file
+weather_effect = {'облачно с прояснениями': 'cloudy with clearings',
+                  'сильный дождь': 'heavy rain',
+                  'переменная облачность': 'partly cloudy',
+                  'дождь': 'rain',
+                  'возможны грозы': 'thunderstorms possible',
+                  'мелкий дождь': 'light rain',
+                  'небольшая облачность': 'partly cloudy',
+                  'сплошная облачность': 'overcast',
+                  }
\ No newline at end of file
Index: users.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>{\r\n    \"508750309\": {\r\n        \"city\": \"марокко\",\r\n        \"notification\": false\r\n    },\r\n    \"484014640\": {\r\n        \"city\": \"Киев\",\r\n        \"notification\": true\r\n    },\r\n    \"410193945\": {\r\n        \"city\": \"Днепр\",\r\n        \"notification\": false\r\n    },\r\n    \"874830971\": {\r\n        \"city\": \"Киев\",\r\n        \"notification\": false\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/users.json b/users.json
--- a/users.json	(revision 5b64cb32cf9887b1febb528b5df26a9bc3a987d4)
+++ b/users.json	(date 1660154736169)
@@ -1,7 +1,7 @@
 {
     "508750309": {
         "city": "марокко",
-        "notification": false
+        "notification": true
     },
     "484014640": {
         "city": "Киев",
